Makefile

```diff
@@ -116,7 +119,7 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 
 .PHONY: uninstall
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | oc delete -f -
+	$(KUSTOMIZE) build config/crd | oc delete --IGNORE-NOT-FOUND=$(IGNORE-NOT-FOUND) -f -
```

Comment:
The flag should be `--ignore-not-found` otherwise i'm not sure oc will recognize it as a valid option